scripts/run-ci-build-wheel.sh

```diff
@@ -33,12 +33,6 @@ FLASHINFER_LOCAL_VERSION="cu${CUDA_MAJOR}${CUDA_MINOR}torch${FLASHINFER_CI_TORCH
 if [ -n "${FLASHINFER_GIT_SHA}" ]; then
     FLASHINFER_LOCAL_VERSION="${FLASHINFER_GIT_SHA}.${FLASHINFER_LOCAL_VERSION}"
 fi
-# wgmma work for cuda 12.3 and above
```

Comment:
where do we determine `FLASHINFER_ENABLE_SM90` after removing these lines?

Reply:
I searched the code base, there's no use of `FLASHINFER_ENABLE_SM90` other than the ones deleted in this PR.

Reply:
We used to rely on this logic to avoid compiling sm90a wheels for cuda 12.1 (wgmma support is started from cuda 12.3), but now I suppose we can safely retire cuda 12.1 wheels as pytorch's cuda 12.1 wheels ended with torch 2.5